src/components/molecules/ZeroResults/index.tsx

```diff
@@ -0,0 +1,32 @@
+import { FC } from "react";
+import { ZeroResultsProps } from "./interface";
+
+import * as S from "./styles";
+
+const ZeroResults: FC<ZeroResultsProps> = ({ setTag, setSearchTerm }) => {
+  const suggestions = ["Ciągi", "Jedzenie", "Zwierzęta"];
+
+  const handleOnClick = (el: string) => {
+    setTag(el);
+    setSearchTerm(el);
```

Comment:
We  could use one style of writing **element** We can choose which one. I don't have favourite one :)

Reply:
The other one would be `element`? Actually here I'd say that we have `tag` or some sort of `tagTitle` simply ;)

Reply:
Ok, I've changed `el` to `tag`.